### DISCUSSION TITLE
Nomination of @cweitkamp

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-core#3494] [Core Maintainers Discussions]
As mentioned [here](https://github.com/orgs/openhab/teams/core-maintainers/discussions/1), I would like to increase the maintainer team of openhab-core and nominate @cweitkamp to be a maintainer of openhab-core. Christoph did [a lot of valuable contributions to the ESH core](https://github.com/eclipse/smarthome/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+author%3Acweitkamp+%5Bcore%5D+) and implemented many features that were strongly desired. The code quality has always been excellent.

**Comment by kaikreuzer** [comment]
Welcome, @cweitkamp, I have added you to the team!

**Comment by cweitkamp** [comment]
Thank you very much.

I'm interested in picking up a new feature for the core framework. I'll open an issue for discussion tomorrow. Topic will be: How to expose Thing / Channel configuration to make it accessible in Rules.
